docs/src/components/Demo.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from 'react';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import Highlight, { defaultProps } from 'prism-react-renderer';
+import { Tabs, TabItem, Card, Flex, View, Button } from '@aws-amplify/ui-react';
+
+export const Demo = ({ children, propControls, themeControls, code }) => {
+  const [copied, setCopied] = useState(false);
+
+  const copy = () => {
+    setCopied(true);
+    setTimeout(() => {
+      setCopied(false);
```

Comment:
Love the UX of this. Clear signal that it was copied, but still lets you copy again if you want later on